notNeededPackages.json

```diff
@@ -904,6 +904,10 @@
             "libraryName": "cnpj",
             "asOfVersion": "3.1.0"
         },
+        "cobe": {
+            "libraryName": "cobe",
+            "asOfVersion": "0.5.0"
```

Comment:
Probably 0.4.4 would have worked; it just needs to be greater than the latest DT published version (for some reason)

But this probably also works ok

Reply:
The CI complained when I tried because 0.4.4 isn't actually a release on npm

Reply:
Can't win! Honestly, I don't understand our restrictions for this

Reply:
IMO it might make sense to either only consider the major/minor versions (eg. making it okay to specify `0.4.2` for `asOfVersion` because it equals the typings' minor version), or to allow non-existent patch versions for cases like this (since `@types` patch versions have no relation to the original packages')

Reply:
It seams typescript support was added in version 0.4.2. Please update the version